scripts/output-intermediate-gitlog.sh

```diff
@@ -5,16 +5,16 @@
 my_dir="$(dirname "$0")"
 cd $my_dir
 
-source "colors.sh"
+source "gitlogg-utils.sh"
 
 cd ..
 
 test "$1" || exit 1
 dir=$1
 
-cd $dir &&
-        echo -e "${Whi}Outputting ${Pur}${PWD##*/}${RCol}" >&2 &&
-        git log --all --no-merges --shortstat --reverse --pretty=format:'commits\trepository\t'"${PWD##*/}"'\tcommit_hash\t%H\tcommit_hash_abbreviated\t%h\ttree_hash\t%T\ttree_hash_abbreviated\t%t\tparent_hashes\t%P\tparent_hashes_abbreviated\t%p\tauthor_name\t%an\tauthor_name_mailmap\t%aN\tauthor_email\t%ae\tauthor_email_mailmap\t%aE\tauthor_date\t%ad\tauthor_date_RFC2822\t%aD\tauthor_date_relative\t%ar\tauthor_date_unix_timestamp\t%at\tauthor_date_iso_8601\t%ai\tauthor_date_iso_8601_strict\t%aI\tcommitter_name\t%cn\tcommitter_name_mailmap\t%cN\tcommitter_email\t%ce\tcommitter_email_mailmap\t%cE\tcommitter_date\t%cd\tcommitter_date_RFC2822\t%cD\tcommitter_date_relative\t%cr\tcommitter_date_unix_timestamp\t%ct\tcommitter_date_iso_8601\t%ci\tcommitter_date_iso_8601_strict\t%cI\tref_names\t%d\tref_names_no_wrapping\t%D\tencoding\t%e\tsubject\t%s\tsubject_sanitized\t%f\tcommit_notes\t%N\tstats\t' |
+tempOutputFile="/home/mbeller/gitloggg/inventitech-gitlogg/_tmp/gitlogg.tmp"
```

Comment:
I deleted this line while reviewing, but still got the error I posted in the PR.